content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Quantity
+id: quantity
+date: 2023-01-24
+short_description: >
+ Quantity contains GB and GiB. 
+
+aka: 
+tags:
+- fundamental
+---
+
+"Quantity" refers to a specific amount of a resource, such as CPU or memory, that is allocated to a container or pod. 
```

Comment:
```suggestion
Quantity refers to a specific amount of a resource, such as CPU or memory, that is allocated to a Pod in which a container is running. 
```
Taking reference from [here](https://kubernetes.io/docs/contribute/style/style-guide/#use-italics-to-define-or-introduce-new-terms)